addons/website_event_exhibitor/controllers/exhibitor.py

```diff
@@ -110,6 +110,7 @@ def _event_exhibitors_get_values(self, event, **searches):
             'hide_sponsors': True,
             # search information
             'searches': searches,
+            'search_count': len(sorted_sponsors),
```

Comment:
@thsh-odoo Hello again, LGTM. :)
Can you just add a small comment in your commit to why you change the name?
something like: _A variable is renamed because it is needed but redefined in the code above the returned dict._ 
I'm greening in the meantime :)